threat-miner/info.json

```
@@ -0,0 +1,278 @@
+                    "editable": true,
+                    "value": "google.com",
+                    "placeholder": "e.g: google.com",
+                    "description": "Domain name required for retrieving malware samples associated with the domain.",
+                    "tooltip": "Domain name required for retrieving malware samples associated with the domain."
+                },
+                {
+					"title": "Query Type",
+					"type": "select",
+					"name": "query_type",
+					"required": true,
+					"visible": true,
+					"editable": true,
+                    "description": "Select the type of query that is how you want to perform get domain operation.",
+                    "tooltip": "Select the type of query that is how you want to perform get domain operation.",
+					"options": [
+						"WHOIS",
+						"Passive DNS",
+						"Example Query URI",
+                        "Related Samples (hash only)",
+						"Subdomains",
+						"Report tagging"
+					]
+				}
+            ]
+        },
+        {
+            "title": "Get IP Details",
+            "operation": "get_ip_details",
+            "category": "investigation",
+            "description": "Returns malware samples associated with an IP address based on the query performed.",
+            "annotation": "ip_details",
+            "output_schema": [
+                {
+                    "status_code": "",
+                    "status_message": "",
+                    "results": []
+                }
+            ],
+            "parameters": [
+                {
+                    "title": "IP Address",
+                    "type": "text",
+                    "name": "ip_address",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "216.58.213.110",
+                    "placeholder": "e.g: 216.58.213.110",
+                    "description": "Required IP address whose all related details needs to be retrieved.",
+                    "tooltip": "Required IP address whose all related details needs to be retrieved."
+                },
+                {
+					"title": "Query Type",
+					"type": "select",
+					"name": "query_type",
+					"required": true,
+					"visible": true,
+					"editable": true,
+                    "description": "Select the type of query that is how you want to perform get domain operation.",
+                    "tooltip": "Select the type of query that is how you want to perform get domain operation.",
+					"options": [
+						"WHOIS",
+						"Passive DNS",
+                        "Related Samples (hash only)",
+						"SSL Certificates (hash only)",
+						"Report tagging"
+					]
+				}
+            ]
+        },
+        {
+            "title": "Get Samples",
+            "operation": "get_file_details",
+            "category": "investigation",
+            "description": "Returns known metadata of the given samples in the form of a note.",
```

```suggestion
            "description": "Returns known metadata of the given samples in the form of a note.",
```
is description correct?